app/presenters/project_dashboard_presenter.rb

```diff
@@ -40,4 +40,13 @@ def role(user)
       "Data User"
     end
   end
+
+  def quota_usage(session_id:)
+    quota_usage = "#{project.storage_usage(session_id:)} out of #{project.storage_capacity(session_id:)} used"
+    quota_usage
+  end
+
+  def quota_percentage(session_id:)
+    # figure out how to calculate percentage of storage used with different and unpredictable units
```

Comment:
I think storage is now a hash.  I wonder if there needs to be some updates to this code.  https://github.com/pulibrary/tigerdata-app/blob/main/app/models/project_metadata.rb#L149